src/duckstation-qt/gamecheatsettingswidget.cpp

```diff
@@ -529,6 +529,7 @@ void GameCheatSettingsWidget::importCodes(const std::string& file_contents)
   }
 
   reloadList();
+  g_emu_thread->reloadCheats(true, true, false, true);
```

Comment:
I'm not sure about this. It shouldn't need to reload cheats, since the one you just imported should not be enabled yet.

Once you click the checkbox, it should trigger a reload.

Reply:
You mentioned on Discord what the bools should be and I can confirm that true, false,false ,true works for both lines.

  g_emu_thread->reloadCheats(true, false, false, true);

I've updated it on my master branch to reflect this.